site/pages/create-disruption.tsx

```diff
@@ -1,9 +1,138 @@
-const CreateDisruption = () => {
-    return <></>;
+import React, { ReactElement } from "react";
+import DateSelector from "components/DateSelector";
+import TimeSelector from "components/TimeSelector";
+import { BaseLayout } from "components/layout/Layout"
+import { DisruptionValidity } from "interfaces";
+
+const title = 'Create Disruptions';
+const description = 'Create Disruptions page for the Create Transport Disruptions Service';
+
+interface CreateDisruptionProps {
+    inputs: DisruptionValidity
+}
+
+const CreateDisruption = ({ inputs }: CreateDisruptionProps): ReactElement => {
+    return (
+        <BaseLayout title={title} description={description}>
+            <>
+                <div className="govuk-form-group">
+                    <fieldset className="govuk-fieldset" role="group" aria-describedby="when-is-the-disruption">
+                        <legend className="govuk-fieldset__legend govuk-fieldset__legend--l">
+                            <h1 className="govuk-fieldset__heading">
+                                When is the disruption?
+                            </h1>
+                        </legend>
+                        <legend className="govuk-fieldset__legend govuk-!-padding-top-4">
+                            What is the start date?
+                        </legend>
+                        <div id="start-date-hint" className="govuk-hint">
```

Comment:
For the hints we should ideally pass the id into the input as an aria-describedby

Reply:
done